framework/auth/core.py

```diff
@@ -286,13 +286,15 @@ class User(GuidStoredObject, AddonModelMixin):
     # }
 
     # email lists to which the user has chosen a subscription setting
-    mailing_lists = fields.DictionaryField()
+    mailchimp_mailing_lists = fields.DictionaryField()
     # Format: {
     #   'list1': True,
     #   'list2: False,
     #    ...
     # }
 
+    osf_mailing_lists = fields.DictionaryField(default={'Open Science Framework Help': True})
```

Comment:
Also, please use the variable in `defaults.py` for the help mailing list instead of the string "Open Science Framework Help"